common/remote/sockfile.cpp

```diff
@@ -2207,8 +2213,10 @@ class CRemoteFileIO : public CInterface, implements IFileIO
         MemoryBuffer replyBuffer;
         const char *localname = parent->queryLocalName();
         localname = skipSpecialPath(localname);
-        // also send _extraFlags
-        sendBuffer.append((RemoteFileCommandType)RFCopenIO).append(localname).append((byte)_mode).append((byte)_compatmode).append((byte)_extraFlags);
+        // also send _extraFlags, eflags
+        unsigned eflags;
+        parent->getCreateFlags(eflags);
```

Comment:
would be better if getCreateFlags returned flags from function rather than filled reference.